.github/ISSUE_TEMPLATE/onboard-new-tenant-test.md

```diff
@@ -0,0 +1,231 @@
+        options:
+         - "No"
+         - "Yes"
+    validations:
+      required: true
+  - type: dropdown
+    id: Product-Areas-Merchants
+    attributes:
+        label: Merchants solution area
+        description: Does you product fall under Merchants category?
+        options:
+         - "Yes"
+         - "No"
+    validations:
+      required: true
+  - type: checkboxes
+    id: customer-segments-for-merchants
+    validations:
+      required: true
+    attributes:
+      label: Customer Segments for Merchants
+      description: Specify the type of segement. Required if you selected Merchants above.
+      options:
+      - label: SMB
+      - label: Enterprise
+  - type: dropdown
+    id: Product-Areas-Financial-Institutions
+    attributes:
+        label: Financial Institutions solution area
+        description: Does your product fall under Financial Institutions category?
+        options:
+         - "Yes"
+         - "No"
+    validations:
+      required: true
+  - type: checkboxes
+    id: customer-segments-for-financial-institutions
+    validations:
+      required: true
+    attributes:
+      label: Customer Segments for Financial Instituitions
+      description: Specify the type of segement. Required if you selected Financial Institutions above.
+      options:
+       - label: Bank
+       - label: Credit Union
+       - label: Large Financial Institution
+  - type: dropdown
+    id: Product-Areas-Fintech
+    attributes:
+        label: Fintech solution area
+        description: Does your product fall under Fintech category?
+        options:
+         - "Yes"
+         - "No"
+    validations:
+      required: true
+  - type: checkboxes
+    id: region-of-operation
+    validations:
+      required: true
+    attributes:
+      label: Region of Operation
+      description: Select your product's operational region. 
+      options:
+       - label: North America
+       - label: EMEA
+       - label: LATAM
+       - label: APAC
+  - type: dropdown
+    id: integration-restful
+    attributes:
+      label: RESTful
+      description: Do you provide restful APIs for integration
+      options:
+        - "Yes"
+    validations:
+      required: true
+  - type: dropdown
+    id: integration-sdk
+    attributes:
+      label: SDK
+      description: Do you provide SDKs for integration
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: textarea
+    id: industry
+    attributes:
+      label: Industry
+      description: Which sectors of industry does your product serve? You may only list up to five different industries. 
+      placeholder: "Examples: Retail, Banking, Petroleum, Airline, Manufacturing"
+  - type: dropdown
+    id: runbox
+    attributes:
+      label: Runbox
+      description: Do you want to enable the RunBox feature? If the preceding response is "Yes," continue filling out the below questions
```

Comment:
Perhaps `If the preceding response is "Yes"` --> `If providing SDKs for integration`?